docs/overview.rst

```diff
@@ -12,12 +12,27 @@ Integration Services architecture
 ---------------------------------
 
 IS provides three interfaces that must be implemented by any bridge that you want to use. This classes are
-:ref:`isbridge`, :ref:`ispublisher` and :ref:`issubscriber`. There is a :ref:`rtps bridge`
+:ref:`isbridge`, :ref:`ispublisher` and :ref:`issubscriber`. There is a :ref:`rtps-bridge`
 implementation as default, that uses Fast-RTPS libraries.
 
-Any :ref:`connector` must have at least one endpoint configured as a Fast-RTPS participant, as IS is intended to communicate
-Fast-RTPS with others protocols when using bridges.
+Any :ref:`connector` must have at least one endpoint configured as a Fast-RTPS participant,
+as IS is intended to communicate Fast-RTPS with others protocols when using bridges.
 
+When you implement your ISBridge derived class, you must take in account:
+
+- Only :class:`ISPublisher::publish` is mandatory to implement.
```

Comment:
Is this true? Why is it mandatory?

Reply:
It's mandatory because that functions are pure virtual. Publish methods will receive data from the bridges and are intended to "publish" or "write" that data to somewhere.
If a publisher does nothing all that data will be lost.